hcx-core/audit-indexer/src/main/resources/audit-mappings.json

```diff
@@ -161,6 +161,9 @@
     },
     "invited_by": {
       "type": "keyword"
+    },
+    "payload_size": {
+      "type": "integer"
```

Comment:
Are we storing the bytes value? Integer might not work. Do we have "number" as datatype. Please check for the limit.